src/lifecycles/prop.helpers.ts

```diff
@@ -26,14 +33,14 @@ export function getProps(appOrParcel) {
       customProps
     );
   }
-  const result = assign({}, customProps, {
+  const result: SingleSpaProps = Object.assign({}, customProps, {
     name,
     mountParcel: mountParcel.bind(appOrParcel),
     singleSpa,
   });
 
   if (isParcel(appOrParcel)) {
```

Comment:
It might be useful to redefine `isParcel()` as:

```ts
function isParcel(appOrParcel: AppOrParcel): appOrParcel is InternalParcel
```

This shouldn't requires any runtime changes, but it eliminates the need for a cast here. Just a thought for a future PR.

Reply:
I wasn't aware of the is `appOrParcel is InternalParcel` syntax - very cool. I've updated it.